README.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Formally verify smart contracts using the tools you already know
+description: Take Kontrol of your smart contracts with Simple Formal Verification
```

Comment:
Agree! I will make the changes as suggested.

Reply:
What do you think about this? I use the color green instead of blue for `easy-to-use formal verification`. I'm not sure if the tags are correct. Where can I test them?

```suggestion
description: Take <ins><font color="#F7CE55">K</font></ins><font color="#F7CE55">ontrol</font> of your smart contracts with <font color="#568A37">easy-to-use formal verification</font>
```

Reply:
It looks like this, I used an online markdown generator: 
<img width="561" alt="image" src="https://github.com/user-attachments/assets/8a1ca3c6-2ce9-4f92-b4a2-d10cf8ff140f">

Reply:
And thanks! The "of" should be removed, but I guess it's fine even if we don't use any colored highlighting :) 

Maybe just "Kontrol your smart contracts with easy-to-use formal verification" in black?

Reply:
What about this?

```suggestion
description: <font color="#F7CE55"><ins>K</ins>ontrol</font> your smart contracts with easy-to-use formal verification
```